app/templates/index.html

```diff
@@ -104,7 +104,7 @@ <h2 id="supplier-message-heading">Sell services</h2>
           {% if logged_in %}
             <p>
               <a href="/suppliers" class="top-level-link">
-                <span>View your services and account details</span>
+                View your services and account details
```

Comment:
We might want a different link for logged-in buyer users, to take them to their buyer dashboard `/buyers` instead.